.circleci/config.yml

```diff
@@ -16,22 +16,21 @@ jobs:
                 name: Create env file
                 command: |
                   touch .env
-                  echo 'POSTGRES_URL=postgres://postgres:example@db.local:5432/network_adequacy\nPOSTGRES_URL_POSTGRES=postgres://postgres:example@db.local:5432/postgres' > .env
+                  echo 'POSTGRES_URL=postgres://postgres:example@db.local:5432/network_adequacy' >> .env
+                  echo 'POSTGRES_URL_POSTGRES=postgres://postgres:example@db.local:5432/postgres' >> .env
+                  echo 'DB_GRACE_PERIOD=20' >> .env
             - run:
                 name: Build containers
-                command: 'docker-compose build backend'
-            # This next step needs to be figured out. The DB initialization needs to happen at a sensible point.
-#            - run:
-#                name: Set up the containerized DB
-#                command: 'make setup-local-db'
+                command: |
+                    docker-compose -f docker-compose.yml -f docker-compose.override.local.yml build backend
             - run:
                 name: Run linters
                 command: |
                     make backend-lint
             - run:
                 name: Run tests
                 command: |
-                    make backend-coverage
+                    docker-compose -f docker-compose.yml -f docker-compose.override.local.yml run backend pytest --cov=backend --cov-config .coveragerc --cov-fail-under=68 --cov-report term-missing
```

Comment:
The issue with this is that the coverage could get different.
Could we have a make coverage that is independant of the docker-compose file

Reply:
Done